content-cache/tests/integration/conftest.py

```diff
@@ -104,8 +104,11 @@ async def deploy_applications_fixture(
                 cert_app_name: model.applications[cert_app_name],
                 metric_app_name: model.applications[metric_app_name],
             }
-        except KeyError:
-            raise RuntimeError("At least one app is missing, you cannot use --no-deploy.")
+        except KeyError as e:
```

Comment:
I think it is team's policy to avoid single character variable names.
So changing `e` to `err` or `exc` might be better.

Reply:
Fixed in 0a8c9a9ff1805b9f994de0746536685b017e7973. Thanks.